README.md

```diff
@@ -225,6 +225,34 @@ to integrate with [ThingsBoard.io](https://thingsboard.io/):
 1. in your `config.ini` assign unique sensor names for your plants
 1. on the ThingsBoard platform create devices and use `Access token` as `Credential type` and the chosen sensor name as token
 
+## Openhab with the internal broker (since v2.4)
+In the following is an example of a textual configuration using the new MQTT plugin from Openhab 2.4 with the internal broker.
+
+### Thing file
+```
+Bridge mqtt:systemBroker:MqttBroker "MQTT Broker" [ brokerid="embedded-mqtt-broker" ]
+{
+    Thing topic FicusBenjamin "Ficus Benjamin"
+    {
+        Channels:
+            Type number : light         "Lichtstärke"           [ stateTopic="miflora/FicusBenjamin", transformationPattern="JSONPATH:$.light" ]
```

Comment:
Could you please use english names so the international community has an easier way of understanding your examples?

Also check the other names in your example.